keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/LocatableSchemaCustomizer.kt

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.netflix.spinnaker.keel.api.Locatable
+import io.swagger.v3.core.converter.ModelConverterContext
+import io.swagger.v3.oas.models.media.Schema
+import kotlin.reflect.full.isSubclassOf
+import org.springframework.stereotype.Component
+
+@Component
+class LocatableSchemaCustomizer : AbstractSchemaCustomizer() {
+  override fun supports(type: Class<*>) = type.isSubclassOf<Locatable<*>>()
+
+  override fun customize(schema: Schema<*>, type: Class<*>, context: ModelConverterContext) {
+    eachSchemaProperty(Locatable<*>::locations) {
+    schema.markOptional(it)
+  }
```

Comment:
nit: indentation.

Reply:
I don't understand why spotless is not catching this